### DISCUSSION TITLE
Keyboard combinations, special keys, grab keyboard events within a WSL application

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/wslg#1280] [Q&A]
Hi all!  Just coming back to WSL, and wow have things gotten great.

question: Is there a way to direct all keyboard events (e.g. alt-tab, windows key, etc....) to the linux app running in wsl and being presented by wslg/mstsc.exe (or is there planned support for this)?  

For example, it would be great to be able to use remmina via WSL to rdp into my various linux boxes, move remmina into fullscreen mode, and have all the "grab keyboard events" icon work so I could use all the special keys and key combinations within the remote desktop be presented by remmina.

I'm not sure if it's possible with the wslg architecture being tuned to display apps through mstsc and not remote desktops.  But it'd be cool.

Also, maybe this is a remmina bug and i'm posting in the wrong place?

Thx!

**Comment by antonkurkin** [comment]
I have the same question
